mne/io/tests/test_reference.py

```diff
@@ -294,8 +308,9 @@ def test_add_reference():
     raw = Raw(fif_fname, preload=True)
     events = read_events(eve_fname)
     picks_eeg = pick_types(raw.info, meg=False, eeg=True)
+    # create epochs in delayed mode, allowing removal of CAR when re-reffing
     epochs = Epochs(raw, events=events, event_id=1, tmin=-0.2, tmax=0.5,
-                    picks=picks_eeg, preload=True)
+                    picks=picks_eeg, preload=True, proj='delayed')
```

Comment:
Pretty sure not: the `delayed` just prevents the constructor from `np.dot`'ing the CAR projector on the data. My choice to raise a `RuntimeError` if `proj['active']==True` should prevent errant ref-additions, partly to ensure other parts of the code don't break.